test/machine_install.py

```diff
@@ -116,7 +116,13 @@ def start(self):
 
         self.http_updates_img_port = self._serve_updates_img()
 
-        iso_path = f"{os.getcwd()}/bots/images/{self.image}"
+        # If custom compose if specified for fetching the image then use that
+        # else get the image from the bots directory
+        compose = os.environ.get("TEST_COMPOSE")
+        if compose:
```

Comment:
Completely optional: You can use a walrus for this:

```py
if compose := os.e.g(...):
    use compose here
```

Reply:
Changed. thanks